### PR TITLE
[SSL] Update certificate authorities info

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
### Summary

PCX-11622
PCX-11971

### CODE REVIEW

File: src/content/docs/ssl/edge-certificates/custom-certificates/index.mdx [hunk 1]
```diff
@@ -19,7 +19,10 @@ When you use custom certificates, the following actions should be considered and
 
 :::note
 
-If your custom certificate does not cover all of your first-level hostnames, you can enable [Universal SSL certificate](/ssl/edge-certificates/universal-ssl/) to cover them. 
+If your custom certificate does not cover all of your first-level hostnames, you can enable [Universal SSL certificate](/ssl/edge-certificates/universal-ssl/) to cover them.
+
+If your custom ceritificate is from a [certificate authority that Cloudflare partners with](/ssl/reference/certificate-authorities/), consider switching to a Cloudflare-managed certificate to benefit from automatic issuance and renewal.
```
Comment (wolruf) [inline src/content/docs/ssl/edge-certificates/custom-certificates/index.mdx]: typo in ceritificate > certificate

Comment (pedrosousa) [reply src/content/docs/ssl/edge-certificates/custom-certificates/index.mdx]:
```suggestion
If your custom certificate is from a [certificate authority that Cloudflare partners with](/ssl/reference/certificate-authorities/), consider switching to a Cloudflare-managed certificate to benefit from automatic issuance and renewal.
```

File: src/content/docs/ssl/reference/certificate-authorities.mdx [hunk 2]
```diff
@@ -94,6 +96,32 @@ You can use the [root CAs list](https://pki.goog/faq/#faq-27) for checking compa
...
+This section summarizes commonly requested client support information. For the complete and most up-to-date certificate compatibility, refer to [SSL.com documentation](https://www.ssl.com/browser_compatibility/).
+
+:::
+
+SSL.com is highly compatible, being accepted by over 99.9% of browsers, tablets, and mobile devices.
+
+SSL.com certificates are [cross-signed with Certum](https://www.ssl.com/repository/) and the [CA that cross-signs intermediates](https://crt.sh/?caid=840) is from 2004.
+
+#### Other resources
+[Acceptable top level domains (TLDs) and current restrictions](https://www.ssl.com/acceptable-top-level-domains-tlds-for-ssl-certificates/)
```
Comment (pedrosousa) [inline src/content/docs/ssl/reference/certificate-authorities.mdx:124]:
```suggestion

[Acceptable top level domains (TLDs) and current restrictions](https://www.ssl.com/acceptable-top-level-domains-tlds-for-ssl-certificates/)
```
Let's add an empty line.

File: src/content/docs/ssl/reference/certificate-authorities.mdx [hunk 3]
```diff
@@ -8,7 +8,9 @@ description: For publicly trusted certificates, Cloudflare partners with
   different certificate authorities (CAs). Refer to this page to check what CAs
   are used for each Cloudflare offering and for more details about the CAs
   features, limitations, and browser compatibility.
-
+banner:
```
Comment (pedrosousa) [inline src/content/docs/ssl/reference/certificate-authorities.mdx]: Hadn't seen this being used yet! 
Maybe we can discuss when we should use it in critique?

**Review by pedrosousa — APPROVED** [review]

**Comment by RebeccaTamachiro** [comment]
Closes #15902

**Pull request merged**